src/CMakeLists.txt

```diff
@@ -239,6 +241,8 @@ install(FILES
 
 install(EXPORT exiv2Config DESTINATION "share/exiv2/cmake")
 
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/exiv2ConfigVersion.cmake DESTINATION "share/exiv2/cmake")
```

Comment:
Could you please change the destination so that we use the `${CMAKE_INSTALL_LIBDIR}` variable?. If you could take the opportunity to also change the line 242 it would be great. In master somebody proposed a patch for fixing that, since it was not working properly for certain linux distribution:

```
install(EXPORT exiv2Config DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/exiv2")

install(FILES ${CMAKE_CURRENT_BINARY_DIR}/exiv2ConfigVersion.cmake DESTINATION "${CMAKE_INSTALL_LIBDIR}/exiv2/cmake")
```

Reply:
Done

Reply:
That's what I call efficiency! +1